pkg/operator/staticpod/controller/prune/prune_controller.go

```diff
@@ -74,7 +76,13 @@ func NewPruneController(
 	}
 	c.retrieveStatusConfigMapOwnerRefsFn = c.createStatusConfigMapOwnerRefs
 
-	return factory.New().WithInformers(operatorClient.Informer()).WithSync(c.sync).ToController("PruneController", eventRecorder)
+	return factory.New().WithInformers(
+		operatorClient.Informer(),
+		// informers are needed here because their Getter are cached lister based
+		kubeInformersForTargetNamespace.Core().V1().Pods().Informer(),
+		kubeInformersForTargetNamespace.Core().V1().ConfigMaps().Informer(),
+		kubeInformersForTargetNamespace.Core().V1().Secrets().Informer(),
```

Comment:
is `secretGetter` even used by this controller ?

Reply:
nope, will remove it